### PR TITLE
completed issue-1-user-can-search-meal - Mohamed 

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-04-14 at 8 34 38 PM
Screen Shot 2021-04-14 at 8 37 57 PM

### CODE REVIEW

**Comment by Mohamed-Elmessiry** [comment]
Screen Shot 2021-04-14 at 9 20 39 PM

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+var searchInput = document.querySelector('.dish-input');
+var button = document.querySelector('.search');
+button.addEventListener('click', handleSearchSubmit);
```
Comment (codingcodymiller) [inline js/main.js]: This should be for the `submit` event

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  text-align: center;
+  height: 266px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.main-two {
+  height: 532px;
+}
```
Comment (codingcodymiller) [inline css/styles.css]: main-one and main-two? What are these for? Try to use names that describe your purpose. You should be using a grid system, I recommend grabbing the `grid.css` file from your solution for `css-grid-classes`, and use that grid.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Mohamed-Elmessiry** [comment]
Screen Shot 2021-04-15 at 1 30 33 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**